### PR TITLE
fix for windows high contrast checkbox issue

### LINKED ISSUE #8334: Check Mark is not visible inside the checkbox in the users section of the Admin in Windows High Contrast Mode
### Issue Summary

On the Admin Interface of the Wagtail website with the url point - /admin/users. While we click on the checkbox near the user details one can find a check mark in the checkbox, as attached below in the screenshot
Screenshot (274)

While on using the Windows High contrast mode the check box doesnt appear as shown in the  screenshot
Screenshot (273)

### Steps to Reproduce

1. Go to the wagtail admin and click on setting and the click on users
2. When you hover on the extreme left of the user details you will be able to click on the box to get it checked 
3. Now turn on the Windows High Contrast mode
4. The checkbox will not have a visible checkmark

### Technical details

* Python version: 3.10.4
* Django version: 4.0.3
* Wagtail version: 3.0a0

### Related PR 
[#7573 Bulk action](https://github.com/wagtail/wagtail/pull/7573)
[#7605 Fix checkboxes appearing always checked in Windows high contrast mode](https://github.com/wagtail/wagtail/pull/7605)

### PR DESCRIPTION
**Pull request by anujaraj**
The PR fixes #8334

_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: client/scss/elements/_forms.scss [hunk 1]
```diff
@@ -149,6 +149,7 @@ input[type='checkbox'] {
   cursor: pointer;
   border: 0;
   padding: 0;
+
```
Comment (lb-) [inline client/scss/elements/_forms.scss:152]: Best to remove any accidental whitespace changes like this. 
```suggestion
```

File: client/scss/elements/_forms.scss [hunk 2]
```diff
@@ -223,6 +224,8 @@ input[type='checkbox']:checked:before {
   // stylelint-disable-next-line property-disallowed-list
   left: 2px;
   inset-inline-start: 2px;
+  filter:contrast(0);
+  fill:currentColor;
```
Comment (lb-) [inline client/scss/elements/_forms.scss:228]: This looks like prettier has not run - remember to run `npm run format` after changing frontend code.

File: client/scss/elements/_forms.scss [hunk 3]
```diff
@@ -250,3 +253,5 @@ textarea,
 select {
   @include transition(background-color 0.2s ease);
 }
+
+
```
Comment (lb-) [inline client/scss/elements/_forms.scss:257]: Same here - accidental whitespace addition

File: client/scss/elements/_forms.scss [hunk 4]
```diff
@@ -223,6 +224,8 @@ input[type='checkbox']:checked:before {
   // stylelint-disable-next-line property-disallowed-list
   left: 2px;
   inset-inline-start: 2px;
```
Comment (lb-) [inline client/scss/elements/_forms.scss:226]: Check this looks ok with non-high contrast mode.

Comment (anujaraj) [reply client/scss/elements/_forms.scss:226]: I think it looks little faded with non high contrast, now if i notice. But I am unable to find any alternative

File: client/scss/elements/_forms.scss (hunk 2 above)
Comment (anujaraj) [reply client/scss/elements/_forms.scss:228]: Ok. I will keep this in mind. Thank you for poiting it out :)

**Comment by lb-** [comment]
Updated title so it's clear what this if for

**Pull request closed without merging**